lib/ex.coffee

```diff
@@ -199,9 +209,9 @@ class Ex
   substitute: (range, args) ->
     args = args.trimLeft()
     delim = args[0]
-    if /[a-z1-9\\"|]/i.test(delim)
+    if /[a-z]/i.test(delim)
```

Comment:
I did'nt make this XD, trust me, I just create the :waq command as you can see here
http://vimdoc.sourceforge.net/htmldoc/editing.html